x-pack/plugins/reporting/server/browsers/default_browser.js

```diff
@@ -4,30 +4,28 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-// import getosSync from 'getos';
-// import { promisify } from 'bluebird';
+import getosSync from 'getos';
+import { promisify } from 'bluebird';
+import { PHANTOM, CHROMIUM } from './browser_types';
 
-// const getos = promisify(getosSync);
+const getos = promisify(getosSync);
 
-// Chromium is unsupported on RHEL/CentOS before 7.0
-// const distroSupportsChromium = (distro, release) => {
-//   if (distro.toLowerCase() !== 'centos' && distro.toLowerCase () !== 'red hat linux') {
-//     return true;
-//   }
-//   const releaseNumber = parseInt(release, 10);
-//   return releaseNumber >= 7.0;
-// };
+// Chromium is unsupported on RHEL/CentOS before 7.0.
+const distroSupportsChromium = (distro, release) => {
+  if (distro.toLowerCase() !== 'centos' && distro.toLowerCase () !== 'red hat linux') {
+    return true;
+  }
+  const releaseNumber = parseInt(release, 10);
+  return releaseNumber >= 7.0;
```

Comment:
It's a bit odd to see a parseInt followed by a floating point comparison. I know this is JS, and it doesn't matter, but it seems like this should be `>= 7`.